multinet.rst

```diff
@@ -1,12 +1,122 @@
-.. _multinet:
+Using Multinet
+==============
 
-========
-Multinet
-========
-The Multinet API and web application
+This chapter is all about how to make use of a running Multinet system. Along
+the way, you will also learn how Multinet structures data into tables and
+networks, collected in different workspaces, and how Multinet thinks of
+multivariate data and multivariate networks.
 
-.. toctree::
-   :maxdepth: 2
+Getting Started
+---------------
 
-   installation
-   testing
+To begin, you will need access to a running Multinet instance (either the
+official instance hosted at https://multinet.app, or a local instance you have
+built and run; see :ref:`quickstart` for help). We will follow some steps to
+illustrate the basic mode of using the system.
+
+**1. Create a Workspace**
+
+Multinet structures all the data it hosts into *workspaces*. A workspace can
+hold several tables of data, as well as several networks assembled from those
+tables. Think of a workspace as a dedicated area for related data to be stored.
+
+In this tutorial sequence we will be recreating the `Paul Revere dataset
+<https://kieranhealy.org/blog/archives/2013/06/09/using-metadata-to-find-paul-revere/>`_
+as a social network connecting the various Revolutionary War figures to the
+social clubs they belonged to.
+
+To create a workspace to hold the materials for this dataset, use the ``NEW
+WORKSPACE`` button appearing near the top of the left sidebar. A dialog box will
+appear asking for the name of the new workspace. Fill this in with ``boston``
+and click on the ``CREATE WORKSPACE`` button. This will create the workspace and
+activate it.
+
+In the main panel, you should see two empty columns, headed by ``Networks`` and
+``Tables`` respectively. These columns will track all of the data that comes to
+live in this workspace.
+
+**2. Upload Node Tables**
+
+Multinet largely thinks of attribute data in tabular terms. That is, Multinet
+can ingest data formatted as (among other thigns) CSV files, storing it as a
+*table* of *rows*, each row containing the same types of attributes in the same
+order. Every node table must have a special attribute called ``_key``, which
+contains a value unique to that row within the table it is part of.
```

Comment:
This won't be true once we've merged Jake's open PR and we have UI to change the key field

Reply:
Yep, we're going to have to get in the habit of updating the documentation as we develop features.